### PR TITLE
Release 14.0.0

### PR DESCRIPTION
**Pull request by camscale**

### CODE REVIEW

**Comment by camscale** [comment]
I think we also need to bump the version in https://github.com/gravitational/teleport/blob/branch/v14/integrations/lib/testing/integration/integration_test.go#L43 - the integrations unit tests are failing because of this - v14 is not less than v14. I guess it becomes v15. Not sure if there's any point bumping the lower version (currently v11). Perhaps on this v14 branch, the min version should be v14?

**Review by r0mant — APPROVED** [review]

**Review by jakule — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,56 +1,303 @@
...
+- Discord and ServiceNow hosted plugins
+- Limited passwordless access for local Windows users in OSS Teleport
+- Machine ID: Kubernetes Secret destination
+
+In addition, this release includes several changes that affect existing
+functionality listed in the “Breaking changes” section below. Users are advised
+to review them before upgrading.
 
-### Breaking Changes
+### Description
```
Comment (zmb3) [inline CHANGELOG.md:24]:
```suggestion
### New features
```

Take it or leave it, no strong preference

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,56 +1,303 @@
...
+By default, Teleport will accept the PROXY line but will prevent connections
+with IP pinning enabled. IP pinning users will need to explicitly enable/disable
+proxy protocol like explained above.
 
-#### DynamoDB Billing Mode
+See more details in our [documentation](docs/pages/management/security/proxy-protocol.mdx).
+
+#### Legacy deb/rpm package repositories are deprecated
+
+Starting from the next major release Teleport 15, legacy package repositories
```
Comment (zmb3) [inline CHANGELOG.md:160]: I would say something like:

Teleport 14 will be the last release published to the legacy package repositories at deb.releases.teleport.dev and rpm.releases.teleport.dev. Starting with Teleport 15, packages will only be published to the new repositories at apt.releases.teleport.dev and yum.releases.teleport.dev.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,56 +1,303 @@
...
+`deb.releases.teleport.dev` and `rpm.releases.teleport.dev` will stop receiving
+new releases.
+
+All users are recommended to switch to `apt.releases.teleport.dev` and
+`yum.releases.teleport.dev` repositories as described in installation
+[instructions](docs/pages/installation.mdx).
+
+#### `Cf-Access-Token` header no longer included with app access requests
+
+Starting from Teleport 14, `Cf-Access-Token` header containing signed JWT token
```
Comment (zmb3) [inline CHANGELOG.md:170]:
```suggestion
Starting from Teleport 14, the `Cf-Access-Token` header containing the signed JWT token
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,56 +1,303 @@
...
+will no longer be included by default with all app access requests. All requests
+will still include `Teleport-JWT-Assertion` containing the JWT token.
+
+See documentation for details on how to inject the JWT token into any header
+using headers [rewriting configuration](docs/pages/application-access/jwt/introduction.mdx#inject-jwt).
```
Comment (zmb3) [inline CHANGELOG.md:175]:
```suggestion
using [header rewriting](docs/pages/application-access/jwt/introduction.mdx#inject-jwt).
```

**Review by zmb3 — APPROVED** [review]

**Review by russjones — APPROVED** [review]

**Review by tigrato — APPROVED** [review]

**Pull request merged**